external/loaders/loaders/mappers/_nudged/_nudged.py

```diff
@@ -207,12 +216,22 @@ def open_nudge_to_fine_multiple_datasets(
 
 
 def _get_datasets(
-    url: str, sources: Sequence[str], consolidated: bool = True
+    url: str,
+    sources: Sequence[str],
+    consolidated: bool = True,
+    cache_size_mb: float = None,
```

Comment:
Should have an `Optional` type hint, or you can make this required as it's a local function and you're always passing this argument.